codalab/rest/bundles.py

```diff
@@ -426,6 +425,67 @@ def _fetch_locations():
     return dict(data=uuids_to_locations)
 
 
+@get('/bundle_stores/', apply=AuthenticatedProtectedPlugin())
```

Comment:
Maybe remove the trailing slash to be consistent 
```suggestion
@get('/bundle_stores', apply=AuthenticatedProtectedPlugin())
```

Do this for all the new endpoints

Also test to see if `/bundle_stores` and `/bundle_stores/` both work after you remove the trailing slash.

Reply:
done